README.md

```diff
@@ -81,7 +88,11 @@ firstLetters('iZir&qym sMdUk drIXw AmIk ]') // => gsdA
 ```js
 firstLetters('iZir&qym sMdUk* drIXw AmIk* ]', false) // => Zsda
 ```
-### toAscii(text) ⇒ <code>String</code>
+**Example** *(ASCII first letters with vishraams)*  
+```js
+firstLetters('iZir&qym sMdUk* drIXw AmIk* ]', true, true) // => Zsda
```

Comment:
Not the best example lol

Reply:
Oh, copied it and forgot to change it lol

Reply:
Done